apps/opik-backend/src/main/java/com/comet/opik/api/resources/v1/priv/DatasetsResource.java

```diff
@@ -98,7 +98,13 @@ public class DatasetsResource {
     @JsonView(Dataset.View.Public.class)
     public Response getDatasetById(@PathParam("id") UUID id) {
 
-        return Response.ok().entity(service.findById(id)).build();
+        String workspaceId = requestContext.get().getWorkspaceId();
+
+        log.info("Finding dataset by id '{}' on workspaceId '{}'", id, workspaceId);
```

Comment:
A general Q: what is the default log level of the app? If we plan to log INFO level by default, do we have aproxiamate expectation of the logs volume that may be generated. If the volume is too big, can make sense to consider log.debug instead

Reply:
Hi @LifeXplorer,

Yeah, the default is INFO; for now, we only add logs at the API level and for expected errors. I believe this is the minimum to allow us to troubleshoot in case of errors or unexpected behaviors. We are talking about 3 or 4 logs per request, which seems reasonable. We may add more debug-level logs soon, but for information, I believe those are relevant.